src/Cameras/arcRotateCamera.ts

```diff
@@ -66,6 +67,17 @@ export class ArcRotateCamera extends TargetCamera {
         this.setTarget(value);
     }
 
+    /**
+     * Defines the target mesh of the camera.
+     * The camera looks towards it form the radius distance.
+     */
+    public get targetHost(): AbstractMesh {

```

Comment:
Why is this not `Nullable<AbstractMesh>`?

Reply:
fixed